src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BALL_COUNT = 100;
+    private static final Random random = new Random();
```

Comment:
Random should not be static
```suggestion
    private final Random random = new Random();
```